src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -178,7 +178,7 @@ public function __invoke(Request $request): JsonResponse
 
         return new JsonResponse([
             'status' => 'OK',
-            'more' => !$pager->isLastPage(),
+            'more' => \count($items) > 0 && !$pager->isLastPage(),
```

Comment:
If there are no results in the initial query, the page is 1, and the lastPage is 0. We end returning true when there are not more elements. IMO this could be solved on the persistence bundles, but if not, this is a possible fix.